pkg/storage/store.go

```diff
@@ -103,21 +103,7 @@ func DeleteTask(id int) {
 
 func StartTask(id int) {
 	tasks := loadTasks()
-
-	var task *Task
-
-	var index int
-	for i, t := range tasks.Tasks {
-		if t.Id == id {
-			index = i
-			task = &t
-			break
-		}
-	}
-
-	if task == nil {
-		panic("Task " + string(id) + " does not exist.")
-	}
+	task, index := getTask(id, &tasks)
```

Comment:
We're kind of breaking convention here. It's not ideal to handle a panic from inside the `getTask` method. I know I shipped the same thing already, but now that you're bringing my attention to it, we should really be returning an error out of that method, or just returning `nil` for task and handle that. It's not really the responsibility of the underlying method to `panic`.

Should be something like this:

```
// This could return task, index, error
task, index := getTask(id, &tasks)
if task == (Task{}) {
     // panic
}
```